src/core/binding_observer.ts

```diff
@@ -72,7 +72,7 @@ export class BindingObserver implements ValueListObserverDelegate<Action> {
   }
 
   private disconnectAllActions() {
-    this.bindings.forEach((binding) => this.delegate.bindingDisconnected(binding, true))
+    this.bindings.forEach((binding) => this.delegate.bindingDisconnected(binding))
```

Comment:
The actual "fix" is adding `true` to line 70 above. However, this makes all cases true, so instead the option is removed entirely.